src/feature/relay/router.c

```diff
@@ -1433,6 +1433,47 @@ router_get_advertised_or_port_by_af(const or_options_t *options,
   return port;
 }
 
+/** As router_get_advertised_or_port(), but returns the IPv6 address and
+ *  port. Returns a null address and zero port, if no ORPort is found. */
+void
+router_get_advertised_ipv6_or_ap(const or_options_t *options,
+                                 tor_addr_port_t *ipv6_ap_out)
+{
+  if (BUG(!ipv6_ap_out))
+    return;
```

Comment:
I would use `tor_assert()` here because this is a caller convention setup rather than external input we can't control.

Furthermore, tor will misbehave after this return by accessing stack junk :S.

Reply:
Fixed in 532d9b6, squashed into #1654.

I'll force-push the squashed and merged branch to this PR.